tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -779,8 +780,7 @@ class Converter {
       // skip control nodes
       if (input_name[0] == '^') continue;
       string name = input_name;
-      auto first = name.find_first_of(':');
-      // TODO(aaroey): why removing the colon but not the zero? A bug?
+      auto first = name.find_last_of(':');
```

Comment:
Because we observed that for some graph (like the SSD one in #19619) there are node names which have colons, meaning that the tensor name will have multiple colons, so we need to remove the last part.

Reply:
Should `first` change to `last` here?

Reply:
Done, sorry I missed that.